app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -38,7 +39,8 @@ class ExplorationActivity : InjectableAppCompatActivity(), StopExplorationInterf
     topicId = intent.getStringExtra(EXPLORATION_ACTIVITY_TOPIC_ID_ARGUMENT_KEY)
     storyId = intent.getStringExtra(EXPLORATION_ACTIVITY_STORY_ID_ARGUMENT_KEY)
     explorationId = intent.getStringExtra(EXPLORATION_ACTIVITY_EXPLORATION_ID_ARGUMENT_KEY)
-    explorationActivityPresenter.handleOnCreate(internalProfileId, topicId, storyId, explorationId)
+    backflowId = intent.getStringExtra(EXPLORATION_ACTIVITY_BACKFLOW_ID)
+    explorationActivityPresenter.handleOnCreate(internalProfileId, topicId, storyId, explorationId, backflowId, this)
```

Comment:
`Context` should be passed as first parameter instead of last.

Reply:
Done :)